libeos-payg/util.c

```diff
@@ -263,3 +265,26 @@ payg_should_check_securitylevel (void)
 
   return TRUE;
 }
+
+/**
+ * payg_get_legacy_mode:
+ *
+ * If eospaygd is running in Phase 2 more, returns %TRUE, otherwise returns
+ * %FALSE
+ */
+gboolean payg_get_legacy_mode (void)
+{
+  return payg_legacy_mode;
+}
+
+/**
+ * payg_internal_set_legacy_mode:
+ *
+ * To be called by eospaygd to initialize the value returned by
+ * payg_get_legacy_mode. Should be called once, only on Phase 2
+ * systems.
```

Comment:
Might be worth adding a parenthetical at the end "(where eos-paygd is run from the primary filesystem not the initramfs)" in case someone reading this is confused about what Phase 2 means.